### DISCUSSION TITLE
接口失效了

### DISCUSSION THREAD

**Discussion by ShuaibZyx** [Binaryify/NeteaseCloudMusicApi#1539] [Q&A]
歌曲url和歌曲详情以及一些别的接口（我没测）在昨天好像失效了
,Failed to load resource: the server responded with a status of 400 (Bad Request)

**Comment by Binaryify** [comment]
无法重现
<img width="1896" alt="image" src="https://user-images.githubusercontent.com/12221718/170465522-ed666c28-cfb1-48d6-be95-6b7f9e91df22.png">

**Reply by ShuaibZyx** [reply]
大佬你好，是线上的端口失效了，本地的没有问题😂

**Comment by ShuaibZyx** [comment]
![L~6VU} T$ ZA%X4@G_2TMGY](https://user-images.githubusercontent.com/106308226/170467814-d464f0ee-4160-4d7f-aa30-4d022c705ed3.png)
![image](https://user-images.githubusercontent.com/106308226/170468331-a837191d-5a85-417e-bb9c-198fc2c9750a.png)
![image](https://user-images.githubusercontent.com/106308226/170468431-96602565-3cc6-46fa-808a-d0f50e28123e.png)
暂时就发现这些接口
前天还可以，昨天晚上就不行了，代码没改过，问了群里的好多小伙伴都说访问不到了，我比较菜，麻烦大佬了

**Comment by Binaryify** [comment]
都进群讨论吧，我很难重现

**Comment by Binaryify** [comment]
看看新版本是不是修复好了
